### PR TITLE
PgAggregationPlugin support big number

### PR DESCRIPTION
**Pull request by yoozo**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
In the aggregate query, type conversion causes precision loss.(sum\min\max\average)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] I have tested locally
- [x] I have performed a self review of my changes
- [ ] Updated any relevant documentation
- [ ] Linked to any relevant issues
- [x] I have added tests relevant to my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] My code is up to date with the base branch
- [x] I have updated relevant changelogs. [We suggest using chan](https://github.com/geut/chan/tree/main/packages/chan)

### CODE REVIEW

File: packages/query/src/graphql/graphql.test.ts [hunk 1]
```diff
@@ -143,7 +156,7 @@ describe('GraphqlModule', () => {
     `;
 
     const results = await server.executeOperation({query: GET_META});
-    expect(`${results.errors}`).toEqual(`Cannot query field "fakeMetadata" on type "_Metadata".`);
+    expect(`${results.errors} `).toEqual(`Cannot query field "fakeMetadata" on type "_Metadata". `);
```
Comment (stwiname) [inline packages/query/src/graphql/graphql.test.ts:159]: Can be reverted? Was this something done by the linter?

File: packages/query/src/graphql/graphql.test.ts [hunk 2]
```diff
@@ -176,4 +189,52 @@ describe('GraphqlModule', () => {
 
     expect(fetchedMeta).toMatchObject(mock);
   });
+
+  // sum(price_amount)
+  it('AggregateSpecsPlugin support big number', async () => {
```
Comment (stwiname) [inline packages/query/src/graphql/graphql.test.ts:194]: I think we need a test for number as well to make sure we're not changing the behaviour there

File: packages/query/src/graphql/graphql.test.ts (hunk 1 above)
Comment (yoozo) [reply packages/query/src/graphql/graphql.test.ts:159]: This was probably changed by me accidentally.

File: packages/query/src/graphql/graphql.test.ts (hunk 2 above)
Comment (yoozo) [reply packages/query/src/graphql/graphql.test.ts:194]: Changes have been made.

**Review by stwiname — APPROVED** [review]

**Pull request merged**